packages/babel-plugin-named-asset-import/index.test.js

```diff
@@ -0,0 +1,50 @@
+const pluginTester = require('babel-plugin-tester');
+const namedAssetImport = require('./index');
+
+pluginTester({
+  plugin: namedAssetImport,
+  pluginOptions: {
+    loaderMap: {
+      svg: {
+        ReactComponent: '@svgr/webpack?-prettier,-svgo![path]',
+      },
+    },
+  },
+  pluginName: 'named-asset-import',
+  snapshot: false,
+  tests: {
+    defaultImport: {
+      code: 'import logo from "logo";',
```

Comment:
Is there a reason these don't include the file name? Why not `import logo from 'logo.svg';`?

Reply:
The first 3 tests are testing cases when file extension does NOT match to plugin config (`.svg` in here),
and rest can cover tests when imported file extension matches to config

Reply:
`svgDefaultImport` covers default import of `logo.svg`